### ISSUE TITLE
Add setting: what happens after frame is rendered

### ISSUE THREAD

**Issue by colorpump** [colorpump/interactive-fragment-shader#4]
### nothing

show only finished frame in Texture output

### Change uniforms, and rerender (Render animation)

You can for example just update a time uniform and do some animations.

### ping-pong

swap input and output texture, in order to get certain effect chains (pipeline) or animations.

**Info:** Ping-pong is a common fragment shader technique: [read this](http://www.pixelnerve.com/v/2010/07/20/pingpong-technique/) if you've no idea what it is, or why it's been used.

**TL;DR** In the parallel shader world you cannot write in the input texture. In order to continue next frame with the result of the previous frame, you have to ping-pong two textures.
